runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathIndicatorPlugin.java

```diff
@@ -230,7 +228,26 @@ private void onPostItemDefinition(PostItemDefinition def)
 
 	private void onPlayerDeath(PlayerDeath death)
 	{
-		newBoneFor(death.getPlayer());
+		if (death.getPlayer() != client.getLocalPlayer())
+		{
+			newBoneFor(death.getPlayer());
+			return;
+		}
+
+		if (client.isInInstancedRegion())
```

Comment:
I don't think we want to add bones in instances for other players either.

Could you pretty please also not add one if player is in region 13362? Duel arena and bones don;t go well togetehr

Reply:
Done and done